src/plugins/gc-subway/page2-fr.hbs

```diff
@@ -0,0 +1,58 @@
+---
+{
+	"title": "[Page 1] - GC menu par carte de métro (provisoire)",
+	"language": "fr",
+	"altLangPrefix": "page2",
+	"pageclass": "cnt-wdth-lmtd",
+	"noContentTitle": true,
+	"breadcrumb": [
+		{ "title": "Canada.ca", "link": "https://www.canada.ca/fr.html" }
+	],
+	"secondlevel": false,
+	"dateModified": "2020-12-07",
+	"share": "true"
+}
+---
+{{>alert-provisional-demos}}
+
+<nav class="provisional gc-subway">
+	<h1 id="gc-document-nav">Gabarit d'initiation de service</h1>
```

Comment:
Can we replace **Gabarit d'initiation de service** by **Composante de navigation de style métro**?